README

```diff
@@ -107,14 +107,14 @@ lib/include directory. There are also test programs in the test/
 directory which give examples of use.
 
 Python bindings are in bindings/python and are commented in a similar
-manner. There is also test code in the test/ directory which give
+manner. There is also a test code in the test/ directory which gives
 examples of use.
 
 Can I help?
 ------------------------------------------------------------------------
 
-libsoc is Open Source Software and as such you are welcome to browse the
-code and either add features, or fix bugs. Please submit pull requests,
+libsoc is an Open Source Software and as such you are welcome to browse
```

Comment:
'libsoc is open-source software' would probably be more gramatically correct if you want to switch to it